lib/modules/dosomething/dosomething_reportback/dosomething_reportback.install

```diff
@@ -92,9 +92,13 @@ function dosomething_reportback_schema() {
       ),
     ),
     'primary key' => array('rbid'),
+    'unique keys' => array(
+      'rbid-uid-nid-run_nid' => array('rbid', 'uid', 'nid', 'run_nid'),
```

Comment:
I think it may require it:

```
SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '1702838-1227-0' for key 'uid-nid-run_nid'
```